libmachine/persist/filestore_test.go

```diff
@@ -26,8 +25,6 @@ func getTestStore() Filestore {
 		os.Exit(1)
 	}
 
-	mcndirs.BaseDir = tmpDir
```

Comment:
not used?

Reply:
@gbraad this is part of test and we are anyway not executing any tests as part of CI, may be better to add a CI which atleast run unit test which exist here?